### DISCUSSION TITLE
Support for Clickhouse backend?

### DISCUSSION THREAD

**Discussion by vijaykramesh** [diesel-rs/diesel#3663] [Ideas]
https://github.com/LemmyNet/lemmy/issues/3005 got me digging a bit into diesel, curious if there is any appetite or anybody already working on adding in a Clickhouse backend?

**Comment by weiznich** [comment]
We do not plan to add another backend to diesel itself as we don't have the capacity for that. Diesel supports implementing custom connections and backends as third party crate. If you are interested in support for a Clickhouse backend I suggest that you go down that route. The documentation of the `Backend` and the `Connection` traits contains some general information on how to approach such an implementation.
